### DISCUSSION TITLE
My styles are gone after install

### DISCUSSION THREAD

**Discussion by hmihail** [wireui/wireui#410] [General]
Hello,

after I add the section 
```
content: [
        ...
        './vendor/wireui/wireui/resources/**/*.blade.php',
        './vendor/wireui/wireui/ts/**/*.ts',
        './vendor/wireui/wireui/src/View/**/*.php'
    ],
```

to my tailwind config file, my styles are gone and my layout becomes a mess. 

Any idea why?
Thanks!

**Comment by ph7jack** [comment]
Does your content path have your project resources?
See a full example [here](https://github.com/wireui/docs/blob/main/tailwind.config.js#L3)
